lib/matplotlib/lines.py

```diff
@@ -204,6 +204,7 @@ def _slice_or_none(in_v, slc):
 @_api.define_aliases({
     "antialiased": ["aa"],
     "color": ["c"],
+    "offcolor": ["oc"],
```

Comment:
I'm not sure this should get an alias. It doesn't seem obvious to me what "oc" means at first glance.

Reply:
Thanks, I had wondered if this is high profile enough to warrant an alias.

Reply:
We can always add one later.